mpsl/CHANGELOG.rst

```diff
@@ -7,15 +7,14 @@ Changelog
    :local:
    :depth: 2
 
-nRF Connect SDK v2.2.0
-**********************
+Main branch
+***********
 
-All the notable changes included in the |NCS| v2.2.0 release are documented in this section.
+All the notable changes included in the main branch are documented in this section.
 
 Bug fixes
 =========
-
-* Fixed an issue where the scheduler could put events in the past (DRGN-17851, DRGN-18105).
+* Fixed an issue where the scheduler could put events in the past. (DRGN-17851, DRGN-18105)
```

Comment:
Isn't this fix in v2.2.0? If so, you can't remove the v2.2.0 header, but must create a new one for Main.